### PR TITLE
Update GCS bucket field description in compute profile to be more accurate

### PR DESCRIPTION
**Pull request by wyzhang**

### CODE REVIEW

**Review by fernst — APPROVED** [review]
LGTM

File: cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json [hunk 1]
```diff
@@ -463,7 +463,7 @@
           "widget-type": "textbox",
           "label": "GCS Bucket",
           "name": "gcsBucket",
-          "description": "The Cloud Storage bucket used by Cloud Dataproc to read/write cluster and job data",
+          "description": "GCS bucket used by Dataproc to stage job dependencies, config files etc",
```
Comment (avirmani) [inline cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json]: Should it say 'Google Cloud Dataproc' ? We can omit 'etc'

**Review by seanzhougoogle — APPROVED** [review]

File: cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json (hunk 1 above)
Comment (wyzhang) [reply cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json]: Done. PTAL

File: cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json [hunk 2]
```diff
@@ -463,7 +463,7 @@
           "widget-type": "textbox",
           "label": "GCS Bucket",
           "name": "gcsBucket",
-          "description": "The Cloud Storage bucket used by Cloud Dataproc to read/write cluster and job data",
+          "description": "Google Cloud Storage bucket used to stage job dependencies and config files for running pipelines in Dataproc",
```
Comment (avirmani) [inline cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json]: "...pipelines in Google Cloud Dataproc"

Comment (wyzhang) [reply cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json]: Done

**Review by avirmani — APPROVED** [review]

**Pull request merged**